airbyte-integrations/connectors/source-mailchimp/build.gradle

```diff
@@ -4,10 +4,41 @@ plugins {
     id 'airbyte-source-test'
 }
 
+// todo (cgardens) - move into airbyte-python.
```

Comment:
this needs to be pushed into a plugin. what i imagine we would do is a `dependOnAirbytePythonLib` plugin. you put in the list of libs you depend on and it handles making sure it can grab the wheel files. 

this might also help us get rid of requirements.txt. tbd. the idea is that we can just `pip install` the wheel files into the virtual env in gradle and then we won't need requirements.txt anymore.

Reply:
right now `build/wheels` contains both the artifact published by the python lib _and_ all of the wheels it depends on. if we move forward with this, i'll split this into two separate dirs.